pkg/apis/stork/v1alpha1/backuplocation.go

```diff
@@ -33,11 +33,12 @@ type BackupLocation struct {
 type BackupLocationItem struct {
 	Type BackupLocationType `json:"type"`
 	// Path is either the bucket or any other path for the backup location
-	Path         string        `json:"path"`
-	S3Config     *S3Config     `json:"s3Config,omitempty"`
-	AzureConfig  *AzureConfig  `json:"azureConfig,omitempty"`
-	GoogleConfig *GoogleConfig `json:"googleConfig,omitempty"`
-	SecretConfig string        `json:"secretConfig"`
+	Path          string        `json:"path"`
+	EncryptionKey string        `json:"encryptionKey"`
```

Comment:
Shouldn't we allow users to specify EncryptionKey from a secret?

Reply:
All parameters can already be provided through a secret